frontend/global/helpers.js

```diff
@@ -1,3 +1,14 @@
+// Polyfills
+
+// String.prototype.replaceAll
+if (!String.prototype.replaceAll) {
+	String.prototype.replaceAll = function (search, replacement) {
+		const target = this;
+		return target.replace(new RegExp(search, 'g'), replacement);
```

Comment:
If we ever want to use a regex to search in replaceAll instead of a string, this won't work. However, the code only uses replaceAll twice and both are with strings to search, so this shouldn't be a problem for us.